android/build.gradle

```diff
@@ -22,6 +22,7 @@ if (project == rootProject) {
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-parcelize'
```

Comment:
We've gotten reports about similar issues from merchants. We should root cause this. Do we need to `api` a dependency in the repo or something?